### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by Hughjanus2022** [TheGameCreators/GameGuruRepo#2986]
Using the structure editor textures keep disappearing,
or complete structure will disappear

**Comment by ArgentArts** [comment]
Possibly this bug:

https://github.com/TheGameCreators/GameGuruRepo/issues/2731

**Comment by Hughjanus2022** [comment]
Not importing anything, just using built in structure editor
textures will go white our disappear completely
My pc MSI GL62M 7RD-056Performance Gaming Laptop Core i7-7700HQ GTX 1050 16GB 1TB

**Comment by Hughjanus2022** [comment]
Yes like in the video but will go white our disappear completely

**Comment by plemsoft** [comment]
@Hughjanus2022 Thanks for reporting this issue, this has been fixed and will be in the next update :)
